.flake8

```diff
@@ -0,0 +1,6 @@
+[flake8]
```

Comment:
you can add the flake8 section into the pyproject.yaml

Reply:
I didn't find that in their docs

Reply:
It doesn't seem to work with either `[flake8]` or `[tool.flake8]`